templates/crate/details.html

```diff
@@ -16,6 +16,13 @@
             <div class="pure-u-1 pure-u-sm-7-24 pure-u-md-5-24">
                 <div class="pure-menu package-menu">
                     <ul class="pure-menu-list">
+                        {%- if details.documented_items and details.total_items -%}
+                            {% set percent = details.documented_items * 100 / details.total_items %}
+                            <li class="pure-menu-heading">Documentation<br>coverage</li>
```

Comment:
This makes it span 1 line instead of two, which makes it easier to read IMO. Plus the whole site is already about documentation so I don't think we have to repeat it everywhere.

```suggestion
                            <li class="pure-menu-heading">Coverage</li>
```

There should also be a corresponding section in `templates/rustdoc/body.html` - there's a lot of duplicated content between those two files I haven't gotten around to refactoring.